firmware/baseband/proc_btlerx.hpp

```diff
@@ -42,48 +43,1931 @@ class BTLERxProcessor : public BasebandProcessor {
     void on_message(const Message* const message) override;
 
    private:
+#define SAMPLE_PER_SYMBOL 1
+#define LEN_DEMOD_BUF_ACCESS 32
+#define DEFAULT_ACCESS_ADDR 0x8E89BED6
+#define NUM_ACCESS_ADDR_BYTE 4
+
+    std::array<std::string, 16> ADV_PDU_TYPE_STR{
```

Comment:
nit: I'm not sure how these are used in the code, but unless you need strings (need allocations, etc) prefer string_view constants instead. There's not runtime allocation initialization.

Pull in the std::literals namespace and use the `"foo"sv` initializer.

Reply:
Removed because didn't need it anyway.